test/functional/autocmd/autocmd_spec.lua

```diff
@@ -34,6 +34,23 @@ describe('autocmds:', function()
     assert.same(expected, eval('g:foo'))
   end)
 
+  it(':close triggers WinClose event', function()
+    command('let g:triggered = 0')
+    command('autocmd WinClosed * :let g:triggered=1')
```

Comment:
It should be `let g:triggered+=1`, so that we can verify that it didn't fire more than once.

Reply:
Fixed that, there was actually a bug in it ;)